README.md

```diff
@@ -1,21 +1,38 @@
-# Simple App Plugin
+# Grafana App Plugin Template
 
-This is just a stub to show how you can create a basic visualization plugin.
+[![Build](https://github.com/grafana/grafana-starter-app/workflows/CI/badge.svg)](https://github.com/grafana/grafana-starter-app/actions?query=workflow%3A%22CI%22)
 
-To work with this plugin run:
-```
-yarn dev
-```
+This template is a starting point for building Grafana App Plugins in Grafana 7.0+
 
-or
-```
-yarn watch
-```
+## Getting started
 
-This will run linting tools and apply prettier fix.
+1. Install dependencies
 
+   ```bash
+   yarn install
+   ```
 
-To build the plugin run:
-```
-yarn build
-```
+2. Build plugin in development mode or run in watch mode
+
+   ```bash
+   yarn dev
+   ```
+
+   or
+
+   ```bash
+   yarn watch
+   ```
```

Comment:
I think this should be brought in line with grafana core (and CRA) across various plug-ins and be `yarn start`.

Reply:
Sounds good. I'll add it to the others as well.